app/views/task/index.html.erb

```diff
@@ -1,26 +1,30 @@
 <body>
+<h1>To Do List</h1>
 <table>
   <tr>
-    <th>Task #</th>
+    <th>Assigned to</th>
     <th>Task Name</th>
-    <th>Date of Completion</th>
+    <th>Completed?</th>
     <th>Delete?</th>
   <tr>
   <% @tasks.to_a.each do |task| %>
   <tr>
-    <td><%= task.id %></td>
+    <td><%= link_to "#{task.person.name}", "/people/show/#{task.person.id}" %></td>
     <td><%= link_to "#{task.name}", "/show/#{task.id}" %></td>
-    <td><% if task.completed_at == nil %>
-       <p class="IP">In Process</p>
+    <td class="task_options"><% if task.completed_at == nil %>
```

Comment:
Per CSS naming conventions - we generally want to use the `-` to separate words in CSS names rather than the `_`